lib/use_cases/administrator/check_if_valid_ip.rb

```diff
@@ -3,14 +3,14 @@ module Administrator
     class CheckIfValidIp
       def execute(address)
         @address = address
-        { success: valid_address? }
+        { success: valid_ipv4_address?, ipv6?: valid_ipv6_address? }
```

Comment:
Could we have an `error_message:` in this object so that its set here rather than in the `ip.rb` model?

It would make it more like the `ActiveRecord` pattern where errors are added to the object.

It would look something like:
```
{ success: valid_ipv4_address?, error_message: whatever_method_generates_the_error_message }
``` 

then the model can just attach the error to the `Ip` object, eg: 
```
errors.add(:address, checker[:error_message])
``` 
which can then pass it up to the view.